components/datadog/agentparams/params.go

```diff
@@ -285,7 +285,7 @@ func WithIntakeHostname(hostname string) func(*Params) error {
 func WithFakeintake(fakeintake *fakeintake.Fakeintake) func(*Params) error {
 	return func(p *Params) error {
 		p.ResourceOptions = append(p.ResourceOptions, utils.PulumiDependsOn(fakeintake))
-		return withIntakeHostname(fakeintake.Host)(p)
+		return withIntakeHostname(fakeintake.Host, fakeintake.Port)(p)
```

Comment:
Good point, testing it

Reply:
You are right, this does not work with ecs. Good catch

Reply:
But this made me realise that `inv aws.create-ecs` does not work without LB, because we assume `https` endpoints on ecs agent params https://github.com/DataDog/test-infra-definitions/blob/pducolin/docker-localhost/components/datadog/agent/ecs.go#L257

Reply:
At the same time we don't support LB on hosts, because we always assume `http` protocol. Creating a card to track those

Reply:
https://datadoghq.atlassian.net/browse/ADXT-769